pwndbg/commands/heap.py

```diff
@@ -80,41 +87,59 @@ def bins(addr=None):
     fastbins = main_arena['fastbinsY']
     bins = main_arena['bins']
 
+    size_t_size = gdb.lookup_type('size_t').sizeof
```

Comment:
Actually, they're not `size_t`, they're pvoid.  This should use `pwndbg.typeinfo.pvoid.sizeof`